blueprints/networking/psc-hybrid/main.tf

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Producer
+
+module "vpc_producer" {
+  source     = "../../../modules/net-vpc"
+  project_id = var.project_id
+  name       = "${var.prefix}-producer"
```

Comment:
nit: wouldn't use prefix for vpcs, as they're namespaced by project and not globally.

Reply:
nit: this avoids clashes with existing resources, I think we often do this in examples

Reply:
Yeah, exactly that was my same thought. I wanted to make it can be easily plugged in environments that may potentially have resources with the same name. I'd be in favor of keeping it, if it's not a problem

Reply:
I would keep it, and also make vpc creation optional and add project-level configuration via the project module (see other comments).